bitnami/spark/values.yaml

```diff
@@ -44,6 +44,23 @@ clusterDomain: cluster.local
 ## @param extraDeploy Array of extra objects to deploy with the release
 ##
 extraDeploy: []
+## @param initScripts Dictionary of init scripts. Evaluated as a template.
+## Specify dictionary of scripts to be run at first boot
+## Alternatively, you can put your scripts under the files/docker-entrypoint-initdb.d directory
+## For example:
+#initScripts:
+  #my_init_script.sh: |
+    #  #!/bin/sh
+   #   touch test
+#echo "Do something." >> test.txt
+initScripts: {}
```

Comment:
I just copied one of the existing examples, the idea, in order to facilitate its usage is put an example where just removing the `##` of the code blocks it works:
```suggestion
## @param initScripts Dictionary of init scripts. Evaluated as a template.
## Specify dictionary of scripts to be run at first boot
## Alternatively, you can put your scripts under the files/docker-entrypoint-initdb.d directory
## For example:
## initScripts:
##   my_init_script.sh: |
##      #!/bin/sh
##      echo "Do something."
##
initScripts: {}
```